src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -208,12 +237,41 @@ class AbstractIntegrationDetailedView<
     );
   }
 
-  //Returns the content shown in the top section of the integration detail
-  renderTopSection() {
+  renderAddInstallButton(hideButton = false) {
```

Comment:
@NisanthanNanthakumar I think `hideButtonIfDisabled` makes more sense because if `disabled =false` then the button is always visible.